docs/components/nav_drawer.md

```diff
@@ -0,0 +1,321 @@
+**Indent** | Within the `Divider` widget use the `indent` parameter
+**End Indent** | Within the `Divider` widget use the `endIndent` parameter
+**Thickness** | Within the `Divider` widget use the `thickness` parameter
+
+### Active text overlay attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Color** | Within the `ListTile` widget use the `selectedTileColor` parameter
+
+
+### Active text attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Text label** | `style` on `title` of the `ListTile` when using a `Text` when the `selected` parameter of the `ListTile` is true
+**Color**      | `style` on `title` of the `ListTile` when using a `Text` when the `selected` parameter of the `ListTile` is true
+**Typography** | `style` on `title` of the `ListTile` when using a `Text` when the `selected` parameter of the `ListTile` is true
+
+### Inactive text attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Text label** | `style` on `title` of the `ListTile` when using a `Text` when the `selected` parameter of the `ListTile` is false
+**Color**      | `style` on `title` of the `ListTile` when using a `Text` when the `selected` parameter of the `ListTile` is false
+**Typography** | `style` on `title` of the `ListTile` when using a `Text` when the `selected` parameter of the `ListTile` is false
+
+### Subtitle attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Text label** | `style` on `subtitle` of the `ListTile` when using a `Text`
+**Color**      | `style` on `subtitle` of the `ListTile` when using a `Text`
+**Typography** | `style` on `subtitle` of the `ListTile` when using a `Text`
+
+### Scrim attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Color**      | Within the `Scaffold` widget use the `drawerScrimColor` parameter
+
+
+## Types
+
+There are three types of navigation drawer, which include 1. [standard](#standard-navigation-drawers) 2. [modal](#modal-navigation-drawers) 3. [bottom](#bottom-navigation-drawers)
+
+![Standard, modal, and bottom navigation drawer examples](assets/nav_drawer/Navigation-drawer_composite.png)
+
+## Standard navigation drawer
+
+[Standard navigation drawers](https://material.io/components/navigation-drawer#standard-drawer) allow interaction with both screen content and the drawer at the same time. They can be used on tablet and desktop, but they aren’t suitable for mobile due to limited screen size.
+
+### Standard navigation drawer example
+
+Standard navigation drawers allow interaction with both screen content and the drawer at the same time. They can be used on tablet and desktop, but they aren’t suitable for mobile due to limited screen size.
+
+_**NOTE: There is no explicit standard drawer in Flutter but it can be built using a `Row` that contains a `Container` and a `Scaffold`, with the `Container` acting as a drawer.**_
```

Comment:
Can you provide a example code and a list of APIs that would simulate a standard nav drawer?